docs/source/python/integration/extending.rst

```diff
@@ -50,12 +50,23 @@ This will not include other parts of the Arrow API, which you will need
 to include yourself (for example ``arrow/api.h``).
 
 When building C extensions that use the Arrow C++ libraries, you must add
-appropriate linker flags. We have provided functions ``pyarrow.get_libraries``
-and ``pyarrow.get_library_dirs`` which return a list of library names and
+appropriate linker flags. We have provided functions ``pa.get_libraries``
+and ``pa.get_library_dirs`` which return a list of library names and
 likely library install locations (if you installed pyarrow with pip or
 conda). These must be included when declaring your C extensions with
 setuptools (see below).
 
+.. note::
+
+   The PyArrow-specific C++ code is now a part of the PyArrow source tree
+   and not Arrow C++. That means the header files and arrow_python library
+   are not necessarily installed in the same location as that of Arrow.
+   To get the new location use the functions mentioned in this section:
```

Comment:
I don't think it's useful to repeat this here, these functions are already mentioned above and below :-)